### ISSUE TITLE
[v1.5] new crash on client side

### ISSUE THREAD

**Issue by xemnes** [jbredwards/Fluidlogged-API#23]
just updated to 1.5 to test it, seems the client now crashes with a new error, crash happens quite late during loading this time

https://pastebin.com/Az134qR3

**Comment by jbredwards** [comment]
can you send the full log?

**Comment by xemnes** [comment]
it was too big for pastebin so had to use discord https://cdn.discordapp.com/attachments/341933983669682185/862724951173496842/message.txt

**Comment by jbredwards** [comment]
seems to be a conflict with the railcraft mod, will be fixed next update

**Comment by xemnes** [comment]
alright, thanks!

**Comment by jbredwards** [comment]
After further testing, I don't think the issue is lies with my mod. When I tested my mod & railcraft without any other mods, the game loaded fine. However when I tried loading it with some other mods, the issue started to occur. I think the issue is caused by a mod removing railcraft's creosote fluid block during load time. Ultimately, I think the issue lies with another mod, but triggered by my mod. 

I've gone through the configs for railcraft, and the creosote fluid block can be disabled there directly (config/railcraft/blocks.cfg/blocks/B:fluid.creosote=false). Disabling that causes the issue to not occur, so I'll leave that as the solution for now.
